attributes/default.rb

```diff
@@ -59,20 +59,27 @@
 case node['platform_family']
 
 when 'rhel'
-  # RHEL5 edge case package set and portmap name
-  if node['platform_version'].to_i <= 5
-    default['nfs']['packages'] = %w(nfs-utils portmap)
-    default['nfs']['service']['portmap'] = 'portmap'
-  elsif node['platform_version'].to_i >= 7
-    default['nfs']['service']['lock'] = 'nfs-lock'
-    default['nfs']['service']['server'] = 'nfs-server'
-    default['nfs']['service']['idmap'] = 'nfs-idmap'
-
-    if node['platform_version'] == '7.0.1406'
-      default['nfs']['client-services'] = %w(nfs-lock.service)
+  case node['platform']
+    when 'amazon'
```

Comment:
@yoshiwaan I am well aware of the root of the issue.  I have even commented in the past that Ohai should not treat Amazon Linux as rhel family since Amazon uses a different version scheme than all other Enterprise Linux platforms.  One cannot determine at a glance what functional level and binary compatibility Amazon Linux has with any other Enterprise Linux platforms in the rhel family classification.